bundles/org.openhab.binding.tibber/README.md

```diff
@@ -0,0 +1,119 @@
+# Tibber Binding
+
+The Tibber Binding connects to the [Tibber API](https://developer.tibber.com), and use queries at frequent polls to retrieve price and consumption information for users with Tibber subscription, and provides additional live measurements via websocket for users having Tibber Pulse. 
+
+Refresh time (poll frequency) is set manually as part of setup, minimum 1 minute.
+
+If using Tibber Pulse, Pulse will be recognized automatically by the binding if associated with HomeID used for setup, and live data stream will be initiated if Pulse verified present. 
+
+## Supported Things
+
+Provided one have Tibber Account, the Tibber API is recognized as a thing in OpenHab using the Tibber binding. Tibber Pulse is optional, but will enable live measurements. The channels (i.e. measurements) associated with the binding: 
```

Comment:
Please begin the sentences on new lines.

OpenHab -> openHAB

Reply:
Corrected.